jquery.pickadate/jquery.pickadate.d.ts

```diff
@@ -99,221 +60,303 @@ interface pickadateOptions extends pickerOptions {
+
+        /**
+         * A majority of the time, the value that needs to be sent to the server is just the hidden value
+         * and not the visible one. To make this happen, use the hiddenName option.
+         * This essentially nullifies the hiddenPrefix and hiddenSuffix, strips the name attribute from the source input, and then sets it as the name of the hidden input:
+         */
+        hiddenName?: boolean;  // default undefined
+
+        /**
+         * By default, typing into the input is disabled by giving it a readOnly attribute.
+         * Setting the editable option to true allows the input field to be edited directly.
+         */
+        editable?: boolean;
+
+        /** Specify where to insert the picker's root element by passing any valid CSS selector to this option */
+        container?: string; // default undefined
+
+        /** The hidden input container */
+        containerHidden?: string; // default undefined
+
+        // Close on a user action
+        closeOnSelect?: boolean; // default true
+        closeOnClear?: boolean; // default true
+
+        // Events
+        onStart?: (event: any) => void;
+        onRender?: (event: any) => void;
+        onOpen?: (event: any) => void;
+        onClose?: (event: any) => void;
+        onSet?: (event: any) => void;
+        onStop?: (event: any) => void;
+    }
 
-    // Disable times
-    disable?: any[]; // arrays formatted as [HOUR,MINUTE] or integers representing hours (from 0 to 23). Switch to whitelist by setting true as the first item in the collection.
+    export interface DateOptions extends Options {
+        // Strings and translations
+        monthsFull?: string[]; // default 'January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'
+        monthsShort?: string[]; // default 'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'
+        weekdaysFull?: string[]; // default 'Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'
+        weekdaysShort?: string[]; // default 'Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'
+        showMonthsShort?: boolean;
+        showWeekdaysFull?: boolean;
+
+        // Buttons
+        today?: string; // default 'Today'
+        close?: string; // default 'Close'
+
+        // Accessibility labels
+        labelMonthNext?: string; // default 'Next month'
+        labelMonthPrev?: string; // default 'Previous month'
+        labelMonthSelect?: string; // default 'Select a month'
+        labelYearSelect?: string; //default 'Select a year'
+
+        /**
+         * True enables the dropdown selector and false replaces it with text.
+         * You can also specify the number of years to show in the dropdown using an even integer
+         * - half before and half after the year in focus. If true the value will default to 10.
+         */
+        selectYears?: boolean|number;
```

Comment:
nitpick, totally up to you: TS compiler source code uses spaces before and after the union operator (`boolean | number`) and IMO it improves readability. [example here](https://github.com/Microsoft/TypeScript/blob/master/bin/typescriptServices.d.ts#L407)